server/events/project_command_runner_test.go

```diff
@@ -14,11 +14,12 @@
 package events_test
 
 import (
-	"github.com/hashicorp/go-version"
-	"github.com/runatlantis/atlantis/server/events/runtime"
 	"os"
 	"testing"
 
+	"github.com/hashicorp/go-version"
+	"github.com/runatlantis/atlantis/server/events/runtime"
+
```

Comment:
extra newline

Reply:
this was added by goimports, using make fmt ??

Reply:
@lkysow  I have left this in ... It's done by make fmt ...

Reply:
Yeah but if you delete it it won't re-add it

Reply:
Cool ok. sorted it now.

Reply:
turns out the newline was already there in master after I've merged with upstream anyways!